.github/workflows/publish.yml

```diff
@@ -470,7 +470,7 @@ jobs:
   publish-docker:
     name: Publish to Docker
     runs-on: ubuntu-20.04
-    timeout-minutes: 15
+    timeout-minutes: 45
```

Comment:
Did you mean to increase this? It seems to take roughly 2 minutes normally so 45 is quite a long timeout.

Reply:
Ah yes, I was looking for an exponential timeout option originally but didn't find one. I will set ``retry_wait_seconds`` to something like 1 min and will revert this back to 15.